plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -91,6 +95,11 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
      */
     private String uid
 
+    /**
+     * Flag to indicate if task belong to an TaskRunArray
+     */
+    private boolean isChild
```

Comment:
Could keep this as an attribute of `TaskRun` is the same vein `isArray` and set when the object is created?

Reply:
Changed in last commit